macros/create_bigquery.sql

```diff
@@ -0,0 +1,219 @@
+{% macro bigquery__create_table_as(temporary, relation, compiled_code, language='sql') -%}
```

Comment:
This is basically a copy and paste of the dbt-utils and just adding some extra logic